00_Core/Scripts/Source/MME_Storage.psc

```diff
@@ -23,11 +22,10 @@ Scriptname MME_Storage Hidden
 function initializeActor(actor akActor, float Level = 0.0, float MilkCnt = 0.0) global
 	Debug.Trace("MME_Storage: Triggered initializeActor() for actor " + akActor.GetLeveledActorBase().GetName())
 
-	if StorageUtil.GetFloatValue(akActor, "MME.MilkMaid.BreastCount") > 0						;for maid<->slave conversion
+	if StorageUtil.HasFloatValue(akActor, "MME.MilkMaid.Level")							;for maid<->slave conversion
```

Comment:
Must now use some other value to identify a maid. Maid level seems the most sensible.